mk/config.mk

```diff
@@ -6,16 +6,16 @@ ifneq ($(SKIP_AUTO_CONF),yes)
 sinclude $(TOPDIR)/autoconf.mk
 endif
 
-AS		= $(CROSS_COMPILE)as
-LD		= $(CROSS_COMPILE)ld
-CC		= $(CROSS_COMPILE)gcc
-CPP		= $(CC) -E
-AR		= $(CROSS_COMPILE)ar
-NM		= $(CROSS_COMPILE)nm
-LDR		= $(CROSS_COMPILE)ldr
-STRIP		= $(CROSS_COMPILE)strip
-OBJCOPY		= $(CROSS_COMPILE)objcopy
-OBJDUMP		= $(CROSS_COMPILE)objdump
+AS		?= $(CROSS_COMPILE)as
+LD		?= $(CROSS_COMPILE)ld
+CC		?= $(CROSS_COMPILE)gcc
```

Comment:
This doesn't work without `-R` in `MAKEFLAGS`, because `CC` is a built-in variable defined by default.

Reply:
you are right, I think this patch is not necessarily needed. We can easly pass CC/CXX etc via make cmdline along with CROSS_COMPILE to override it. So lets drop this patch.